railgun/config/variants/enterprise/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+resources:
+- ../../base/
+patchesStrategicMerge:
+- kong-enterprise.yaml
+images:
+- name: kong
+  newName: kong/kong-gateway
+  newTag: 2.4.1.0-alpine
```

Comment:
please do not hardcode tag, otherwise, we need add duplicate yaml each round.

Reply:
This is following the convention we use for 1.x, where both the Enterprise variants override their image in kustomization.yaml:
https://github.com/Kong/kubernetes-ingress-controller/blob/1.3.1/deploy/manifests/enterprise/kustomization.yaml
https://github.com/Kong/kubernetes-ingress-controller/blob/1.3.1/deploy/manifests/enterprise-k8s/kustomization.yaml

That is an artifact from an earlier era where database-backed and DB-less Enterprise clusters did actually use different images. We probably can now look at making them cascade, where only only the Enterprise DB-less kustomization.yaml overrides the image and the Enterprise Postgres variant builds off it, but it'd require some refactoring to do correctly, since we currently build Enterprise Postgres off of OSS Postgres, not Enterprise DB-less.

Updating the images in each is simple grep and replace, so my vote is to defer that refactor; it should be separate from the work here.